src/Microsoft.AspNetCore.Dispatcher/IDefaultMatcherFactory.cs

```diff
@@ -7,6 +7,6 @@ namespace Microsoft.AspNetCore.Dispatcher
 {
     public interface IDefaultMatcherFactory
```

Comment:
Do we need an interface at all for the default?

Reply:
I introduced it because of a layering issue, but I kinda like it. It won't be necessary when we're done